src/pages/docs/deprecations/index.md

```diff
@@ -56,7 +71,7 @@ For background, migration guidance, and a detailed timeline, please see our blog
 
 ### Defaulting C# scripting to dotnet-script
 
-We announced the deprecation of ScriptCS back in 2022, with pathways for migrating added in 2024. As of 2025.2 we have switched the default C# scripting library over to dotnet-script. A migration guide is available [here](https://g.octopushq.com/ScriptCSDeprecation).
+We announced the deprecation of ScriptCS back in 2022, with pathways for migrating added in 2024. As of 2025.2 we have switched the default C# scripting library over to dotnet-script. A migration guide is available [on our blog](https://g.octopushq.com/ScriptCSDeprecation).
```

Comment:
unrelated? I'm okay with the change, but should it be in this PR? Do we have a new validation rule that triggers on vague link names like "here"?